Nigorjeanluc/Broadcaster
# #169951391 Work on the UI feedback

### What does this PR do?
This pull request is used to re-design the UI in order to achieve better user experience(accessibility, findable buttons, desirability, usable and useful links).

### Description of Task to be completed?
The tasks done here are
- modify the list of cards into a table
- add a page to view a single entry
- add a page to create a new entry

### How should this be manually tested?
Clone or download the develop branch to your local machine and you can see the changes or visit [this link](https://nigorjeanluc.github.io/Broadcaster/UI/html/index.html)

### What are the relevant pivotal tracker stories?
[Finishes #169951391]
